deno/deepgram_audio_summary/README.md

```diff
@@ -0,0 +1,74 @@
+# ⚡ deepgramAudioSummary()
+
+A Deno Cloud Function for transcribing audio using [Deepgram](https://deepgram.com/)
+
+_Example function payload:_
+
+```json
+{
+    {
+        "fileUrl":"https://static.deepgram.com/examples/interview_speech-analytics.wav"
+    }
+}
+```
+
+_Successful function response::_
+
+```json
+{"success":true,"deepgramData":{}}
+```
+
+_Error function response:_
+
+```json
+{"success":false,"message":"Please provide a valid file URL."}
+```
+
+## 📝 Environment Variables
+
+List of environment variables used by this cloud function:
```

Comment:
Would you please remove references to Environment since that's from older runtimes:

```suggestion
## 📝 Variables

List of variables used by this cloud function:
```